tests/codebuild/testfiles/failing-xgboost-mnist-hpo.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: sagemaker.aws.amazon.com/v1
+kind: HyperparameterTuningJob
+metadata:
+  name: failing-xgboost-mnist-hpo
```

Comment:
Can we add comment explaining why we expect this job to fail? Same for `failing-xgboost-mnist`